docs/content/deployment/deploying_scheduler.md

```diff
@@ -36,42 +36,88 @@ steps to initialize configuration and start the Airflow scheduler.
+   ```
+   
+4. Start the Scheduler
+    ```
+    start-airflow.sh
+    ```
 
-3. Prepare the Airflow configuration. To enable CeleryExecutor of the scheduler, you need to run the following command
-   on scheduler server and all celery workers to make sure the configurations among hosts are equivalent. The broker URL
-   and result backend you set up should be passed as well.
+#### Deploy with Docker Compose
 
-   ```bash
-   init-airflow-with-celery-executor.sh $MYSQL_CONN $BROKER_URL $RESULT_BACKEND
-   ```
+Deploying CeleryExecutor is a bit complicated, you can follow this guide to quickly start AIFlow and Scheduler with CeleryExecutor within Docker.
 
-4. Start Celery cluster.
+Note: the Docker Compose below may be not enough to run production-ready Docker Compose AIFlow installation using it.
+This is truly quick-start docker-compose for you to get your hands dirty with AIFlow and CeleryExecutor.
 
-   ```bash
-   # Run following command on every Celery node in cluster, you may see ```celery@node_id ready.``` once worker started successfully.
-   airflow celery worker
-   ```
+##### Before you begin
 
-5. Start the Scheduler
+Make sure you have Docker and Docker Compose installed on your workstation,
+otherwise please follow these steps to install the tools.
 
-   ```bash
-   start-airflow.sh
-   ```
+1. Install [Docker Community Edition (CE)](https://docs.docker.com/engine/install/) on your workstation. Depending on the OS, you may need to configure your Docker instance to use 4.00 GB of memory for all containers to run properly.
+2. Install [Docker Compose](https://docs.docker.com/compose/install/) on your workstation.
+
+Run following commands to check if the tools are installed successfully.
+```shell script
+docker --version
+docker-compose --version
+```
+
+##### docker-compose.yaml
+To deploy AIFlow with CeleryExecutor on Docker Compose, you should fetch [docker-compose.yaml](http://raw.githubusercontent.com/flink-extended/ai-flow/master/docker-compose.yaml).
+
+```shell script
+curl -LfO http://raw.githubusercontent.com/flink-extended/ai-flow/master/docker-compose.yaml
+```
+
+This file contains several service definitions:
+
+* AI Flow servers - AI Flow Server, Notification Server and Scheduler deployed in one container.
+* Workers - The workers who execute the tasks given by the Scheduler.
+* Redis - The broker that forwards messages from Scheduler to worker.
+* Flower -  The [flower app](https://flower.readthedocs.io/en/latest/) for monitoring the environment. It is available at http://localhost:5555.
+* MySQL - The database. Only MySQL is supported in distributed mode for now.
+* HDFS - The hadoop dfs which is used to store the source code or any resources of your projects.
+
+##### Initializing Environment
+
+This command should be executed in the directory that contains docker-compose.yaml you just downloaded.
+It will help start up database, redis, hdfs and 2 workers and also initialize execution environment of all servers.
```

Comment:
It's recommended to tell the users how to check whether the environment has been initialized.